tests/wpt/web-platform-tests/FileAPI/file-upload-frame.html

```diff
@@ -0,0 +1,8 @@
+<form id="testform" enctype="multipart/form-data" action="resource/file-submission.py" method="post">
```

Comment:
This test can't be in wpt, it needs to be under mozilla/

wpt is for tests which can work on any browser; this requires nonstandard `selectFIles`